requirements.txt

```diff
@@ -1,8 +1,7 @@
-typer
-pandas
-requests
-numpy
-semver
-matplotlib
-tabulate
-pydantic
\ No newline at end of file
+pandas<=2.0.3
+requests<=2.31.0
+numpy<=1.24.1
+semver<=3.0.1
+matplotlib<=3.7.2
+tabulate<=0.9.0
+pydantic<=1.10.11
```

Comment:
Do you think we should also use lower boundary for dependencies? Otherwise it is hard to guess what version is used. At leased in other projects I usually see lower boundary as well when upper boundary is used.

Reply:
I think it is a good idea to know the range, but we are still not sure which version will be used until we do a `pip install ...`. Anyway, good suggestion. Resolved in https://github.com/vikinganalytics/mvg/pull/217/commits/855414d0cb0137abb9559f780f1a322e455dd7f9